index.js

```diff
@@ -37,13 +37,18 @@ module.exports = function requireAll(options) {
 
       if (excludeDirectory(file)) return;
 
-      modules[map(file, filepath)] = requireAll({
-        dirname: filepath,
-        filter: filter,
-        excludeDirs: excludeDirs,
-        map: map,
-        resolve: resolve
-      });
+      var directoryModules = requireAll({
+		  dirname: filepath,
```

Comment:
It seems like the intention is messed up in the changes. GitHub is showing the new line significantly more indented than before.

Reply:
sorry, my bad.